styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -3,40 +3,53 @@
   display: block;
 }
 
-.topic .topic_main {
+.topic .topic_main.css-grid {
   display: grid;
+  display: -ms-grid;
   grid-template-columns: 100%;
-  grid-column-gap: 0;
+  padding: 28px 0;
```

Comment:
This seems to behave differently than the `grid-column-gap` property. See screenshot.